_component_design/modal.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Modal
+parent: Content Areas
+layout: component
+category: Components
+usage: Modals to command the user’s attention by obstructing the view of all items not related to the action. Confirmation modals are the most common form and are used when an action could have a destructive result, such as deleting a server or resetting an API key.
+resource: true
+status: unknown
+preview-image: preview-images/modal.svg
+last-modified: 2017-11-01
+---
+
+{% include toc.html %}
+
+## Introduction
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Best practices
+ * Use modals sparingly, only in situations where the user must confirm before continuing, or when the cost of an error could be very high.
+ * It’s best to trigger a modal from a button, link or other clickable item on the page.
+ * Avoid opening modals unexpectedly.
+ * Allow a user to close/cancel a modal by clicking outside of it.
+ * The confirmation button must always reflect the action being confirmed, and adhere to the “Verb Noun” labeling convention.  For example, if the modal title is “Permanently Delete this Server?”, then the confirmation button should be labeled as “Delete Server”.
```

Comment:
Remove comma in first sentence here.

I'd follow the button names and UI names guidance from the [style guide](http://rackerlabs.github.io/docs-rackspace/style-guide/style/text-formatting.html) and bold instead of quote the names:
```markdown
For example, if the modal title is **Permanently delete this server?**, label the confirmation button as **Delete server**.
```

Reply:
👍